src/kixi/datastore/metadatastore.clj

```diff
@@ -52,7 +52,7 @@
   (s/cat :segmentations (s/+ ::segmentation)))
 
 (s/def ::filemetadata
-  (s/keys :req [::type ::id ::name ::schemastore/id ::provenance ::size-bytes]
+  (s/keys :req [::type ::id ::name ::schemastore/name ::provenance ::size-bytes]
```

Comment:
what is the relationship between a `:schemastore/name` and a `:metadatastore/name`?